.flake8

```diff
@@ -3,6 +3,7 @@ min_python_version = 3.6.0
 max-line-length = 88
 ban-relative-imports = true
 inline-quotes = double
+mypy-init-return = True
```

Comment:
Let's add a comment here explaining which plugin this corresponds to / maybe a doc link? Also let's go for a lower case `true` since that's what we've done in the rest of the file.